Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: down deploy shell up
+
+#setup image tag to latest for dev
+IMAGE_TAG?=latest
+DEPLOY_TYPE?=dev
+
+# stop containers
+down:
+	sudo docker compose down
+
+# run the deploy script, passing in deployment type
+deploy:
+	sudo ./deploy.sh -d $(DEPLOY_TYPE)
+
+# exec into api container
+shell:
+	sudo docker compose exec api sh
+
+# (local) development
```

Comment:
My understanding this would be useful for directly running code without having to create tags (or even knowing which is the latest tag). e.g. after cloning the repo and one wants to run `main` or a given `feature` branch **if** `deploy.sh -d dev` doesn't support this.

Is it the case?

Reply:
Not sure I understand you case above @kilemensi , the shell command is/was meant for any debug commands/tasks within the api container

Reply:
My comment is for the `up` command @thepsalmist ... I understood and am happy with the `shell` command.

Reply:
Yes

Reply:
Not sure @thepsalmist ... just had a quick in [`deploy.sh`](https://github.com/mediacloud/story-indexer/blob/3b10677cf4c2e7ef24306e1a5c4993418821f06c/docker/deploy.sh#L50) in story-index repo and:
```sh
    echo "  -a      allow-dirty; no dirty/push checks; no tags applied (for dev)"
```

Do we need multiple ways of doing the same thing?

Reply:
@kilemensi